### PR TITLE
Add additional test coverage for custom Prometheus metrics

### PR DESCRIPTION
**Pull request by sutaakar**

### CODE REVIEW

**Comment by sutaakar** [comment]
Jenkins retest this

**Comment by sutaakar** [comment]
@MarianMacik or @mswiderski Could you please take a look and possibly merge?

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-custom-extension/kie-server-integ-tests-custom-extension-test/src/test/filtered-resources/kjars-sources/cloudbalance/src/main/java/org/kie/server/testing/AbstractPersistable.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.optaplanner.core.api.domain.lookup.PlanningId;
+import org.optaplanner.core.api.score.constraint.ConstraintMatch;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public abstract class AbstractPersistable implements Serializable,
+                                                     Comparable<AbstractPersistable> {
+
+    // TODO Use @XmlID @XmlJavaTypeAdapter(IdAdapter.class) to allow CloudProcess's usage of @XmlIDREF
```
[the remark this answers is not in the document]
Comment (sutaakar) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-custom-extension/kie-server-integ-tests-custom-extension-test/src/test/filtered-resources/kjars-sources/cloudbalance/src/main/java/org/kie/server/testing/AbstractPersistable.java]: Not sure, this is a copy paste from optaplanner Kie server test.

**Comment by MarianMacik** [comment]
jenkins retest this

**Pull request merged**